reporting/report-bl-dest-ips.go

```diff
@@ -35,6 +36,9 @@ func printBLDestIPs(db string, res *resources.Resources) error {
 	if err != nil {
 		return err
 	}
+	if len(w) == 0 {
+		return cli.NewExitError("No results were found for " + db, -1)
```

Comment:
Previously, if there were no results it would show a page with the header menu and an empty table like this:

![image](https://user-images.githubusercontent.com/1696711/67616280-964b8000-f79c-11e9-8ad9-c498b6f204a8.png)

Now, it shows a completely empty page with no menu. I could post a screenshot but it would be pure white :)

I'd like to still get the previous page that had the menu at the top so you could keep clicking through to other modules. This applies to every module.

Beacons seems to be the only module that previously would give an empty white page. So beacons should be changed from its previous behavior to have the menu like the others.